ui/app/pages/swaps/swaps.util.js

```diff
@@ -139,6 +141,24 @@ const AGGREGATOR_METADATA_VALIDATORS = [
   },
 ]
 
+const SWAP_GAS_PRICE_VALIDATOR = [
+  {
+    property: 'SafeGasPrice',
+    type: 'string',
+    validator: (string) => !isNaN(string) && !isNaN(parseFloat(string)),
```

Comment:
Are these three numbers supposed to be decimal or hex? I'm guessing decimal, based on how it's used.

Reply:
These are decimal numbers.

Reply:
Could this be documented somewhere in a comment, or added to the validation, or both? Hex vs. decimal mistakes are not uncommon.

Reply:
Addressed in 0d4389800